### DISCUSSION TITLE
How to filter accoridng to a data in nested response ?

### DISCUSSION THREAD

**Discussion by sambitraze** [directus/directus#12498] [Archive]
https://api.collegeking.in/items/college_College_Events?fields=College_Events_id.*&filter[college_id][_eq]=8

in this, I want to filter out responses whose type = "Workshop".

**Comment by rijkvanzanten** [comment]
See https://docs.directus.io/reference/query/#filter. There's an example at the bottom 🙂 

In your particular case, it'd be 

```
GET /items/college_College_Events
  ?fields=College_Events_id.*
  &filter[college_id][_eq]=8
  &filter[College_Events_id][type][_neq]=Workshops
```

**Reply by sambitraze** [reply]
thanks but i tried this method only it was not working earlier I guess some typo in the parameters
😂😂😂
